fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/PermissiveAvailabilitySpec.groovy

```diff
@@ -49,15 +50,14 @@ class PermissiveAvailabilitySpec extends Specification  {
 
```

Comment:
Basically a column that doesn't participate at all should still overwhelm everything else if it's big in time.